open_tms_backend/open_tms_create_organisation/index.js

```diff
@@ -1,19 +1,73 @@
 import { shared } from '@appblocks/node-sdk'
+import { v4 as uuidv4 } from 'uuid'
 
 const handler = async (event) => {
```

Comment:
Please use examples/list_tickets as reference and add swagger documentation here to the handler so that this can be tested via swagger.Also refer to contribution.md.